homeassistant/components/gree/switch.py

```diff
@@ -12,6 +16,29 @@
 from .const import COORDINATORS, DISPATCH_DEVICE_DISCOVERED, DISPATCHERS, DOMAIN
 from .entity import GreeEntity
 
+GREE_SWITCHES: tuple[SwitchEntityDescription, ...] = (
+    SwitchEntityDescription(
+        icon="mdi:lightbulb",
+        name="Panel Light",
+        key="light",
+    ),
+    SwitchEntityDescription(
+        name="Quiet",
+        key="quiet",
+    ),
+    SwitchEntityDescription(
+        name="Fresh Air",
```

Comment:
This should have been `Fresh air`

Reply:
Hi @frenck 
The name is currently part of the unique-id (see https://github.com/home-assistant/core/pull/90222)
I felt @solazs and/or @jeeftor could/should adjust it in a follow-up PR.

Reply:
> The name is currently part of the unique-id 

That should have been solved first before doing anything else IMHO.